lib/sla.application.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var dal = require('./data/dal.js');
+var stateMachine = require('./core/stateMachine.js');
+
+/**
+ * Builds the application and then initiates the state machine.
+ * @param {Object} client - Object that contains information from the ARI
+ *   connection.
+ * @param {Object} data - the data {trunks, stations, etc.) related to this
+ *   extension.
+ * @param {Object} channel - The incoming channel.
```

Comment:
This is actually a "Channel" object from ari-client. Object doesn't give much information as to what kind of object it is.